src/styl/site.styl

```diff
@@ -70,11 +75,15 @@ body
 
 em, .font-italic
   font-style: italic
+  font-family: $font-sans-italic
 
-strong, .font-bold
-  font-family: $font-bold
+strong
```

Comment:
Do you not think this should map to the 500 weight bold too? Otherwise any user edited font set as bold with be false bolded I think